Source/Core/DolphinQt/SkylanderPortal/SkylanderPortalWindow.cpp

```diff
@@ -296,7 +296,7 @@ QVBoxLayout* SkylanderPortalWindow::CreateFinderLayout()
   auto* search_radio_layout = new QHBoxLayout();
 
   auto* radio_layout_left = new QVBoxLayout();
-  for (int i = 0; i < 10; i += 2)
+  for (int i = 0; i < 12; i += 2)
```

Comment:
```suggestion
  for (size_t i = 0; i < m_element_filter.size(); i += 2)
```

and casting to int in the setProperty call or,

```suggestion
  for (int i = 0; i < std::ssize(m_element_filter); i += 2)
```

will allow you to not need to update this in the future if the size ever changes again